handlers/system-notify.js

```diff
@@ -28,40 +28,15 @@ module.exports = {
     */
    fn: function handler(req, cb) {
       req.log("log_manager.notification");
-      if (process.env.SENTRY_ENABLED) {
```

Comment:
I think there should be other logging strategies in addition to Sentry.  See my note on the `ab_utils` update.